glide.lock

```diff
@@ -1,5 +1,5 @@
-hash: f673b06826e9fa526bda285c5109b133989e188db2d8aca9dbc6913af034857d
-updated: 2019-02-19T14:54:01.742954187-05:00
+hash: 38c293cd395b48a47f66c9434e2e7b1b19aa388c37259a06d772e6ceaca4c1cc
```

Comment:
not sure if this is correct as this `hash` might correspond to the content of `glide.yaml` which might be different for the `master` branch.